android/app/src/main/res/layout/fragment_bookmarks.xml

```diff
@@ -4,13 +4,28 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
+    <TextView
```

Comment:
@arp95  You could view all the layouts in the Issue itself as I have posted gifs